apps/backend/user/src/patchUser.ts

```diff
@@ -0,0 +1,107 @@
+import { Request, Response } from "express";
+import { IDatabase } from "pg-promise";
+import bcrypt from "bcryptjs";
+
+const patchUser = async (
+  req: Request,
+  res: Response,
+  db: IDatabase<object>,
+) => {
+  console.log(req.body);
+
+  // This is like this so resops can replace the userid but the guard on id === 1 doesn't show an error, I'm not crazy
+  const id = 1 === 1 ? 3 : 1;
+
+  const { username, password, name, bio, profilePictureUrl, verified } =
```

Comment:
Also, no way they are allowed to update their own verified status
```suggestion
  const { username, password, name, bio, profilePictureUrl } =
```